### PR TITLE
CSP-105-Web-ConfirmRegulatedStandard

### PR DESCRIPTION
**Pull request by raviudari01**

### CODE REVIEW

File: src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs [hunk 1]
```diff
@@ -81,6 +81,8 @@ public async Task<IActionResult> ViewStandard(int larsCode)
             
             model.EditContactDetailsUrl = Url.RouteUrl(RouteNames.GetCourseContactDetails, new { Ukprn, larsCode });
 
+            model.ConfirmRegulatedStandardUrl = Url.RouteUrl(RouteNames.ConfirmRegulatedStandard, new { Ukprn, larsCode }, Request.Scheme, Request.Host.Value);
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs:84]: What is the purpose of Request.Scheme and Request.Host? Other url seems to be working with out adding these information.

**Review by DevenKShah — CHANGES_REQUESTED** [review]
Half way in review I realised that this has not been merged well, I can see changes from 122 in here

File: src/SFA.DAS.Roatp.CourseManagement.Application/Standard/Commands/UpdateApprovedByRegulator/UpdateApprovedByRegulatorCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+        public async Task<Unit> Handle(UpdateApprovedByRegulatorCommand command, CancellationToken cancellationToken)
+        {
+            _logger.LogInformation("Update confirm regulated standard information request for ukprn:{ukprn} LarsCode:{larscode}", command.Ukprn, command.LarsCode);
+
+            var statusCode = await _apiClient.Post<UpdateApprovedByRegulatorCommand>($"providers/{command.Ukprn}/courses/{command.LarsCode}/update-approved-by-regulator", command);
+
+            if (statusCode != System.Net.HttpStatusCode.NoContent)
+            {
+                _logger.LogError("Failed to confirm regulated standard information request for ukprn:{ukprn} LarsCode:{larscode}", command.Ukprn, command.LarsCode);
```
Comment (MarkFCain) [inline src/SFA.DAS.Roatp.CourseManagement.Application/Standard/Commands/UpdateApprovedByRegulator/UpdateApprovedByRegulatorCommandHandler.cs:30]: similar note to above,, change 'confirm' to 'set approval for '

**Review by MarkFCain — COMMENTED** [review]
I've added a few optional changes to words in logs/exceptions, I'll leave it to you.  I will approve when Deven's issues are resolved

**Review by MarkFCain — APPROVED** [review]

File: src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs [hunk 3]
```diff
@@ -81,7 +79,7 @@ public async Task<IActionResult> ViewStandard(int larsCode)
             
             model.EditContactDetailsUrl = Url.RouteUrl(RouteNames.GetCourseContactDetails, new { Ukprn, larsCode });
 
-            model.ConfirmRegulatedStandardUrl = Url.RouteUrl(RouteNames.ConfirmRegulatedStandard, new { Ukprn, larsCode }, Request.Scheme, Request.Host.Value);
+            model.ConfirmRegulatedStandardUrl = Url.RouteUrl(RouteNames.ConfirmRegulatedStandard, new { Ukprn, larsCode });
```
Comment (DevenKShah) [inline src/SFA.DAS.Roatp.CourseManagement.Web/Controllers/StandardsController.cs:82]: Missed check if IsRegulatedStandard

**Review by DevenKShah — APPROVED** [review]
Minor suggestions

**Pull request merged**